software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcessWinRmDriver.java

```diff
@@ -76,10 +77,13 @@ public void customize() {
 
     @Override
     public void launch() {
-        executeCommandInTask(
```

Comment:
To clarify my suggestion is only for Windows.
From the experience we have with windows, I see only blueprints with install command and no blueprints with launch command.